README.md

```diff
@@ -79,69 +75,9 @@ Exit
 ##### Linux (*Tested only on Ubuntu 14.04*)
 - PowerShell Core
 
-### From Source
-
-* [.NET Core 2.1.4 SDK](https://github.com/dotnet/core/blob/master/release-notes/download-archives/2.0.5-download.md)
-* [PlatyPS 0.9.0 or greater](https://github.com/PowerShell/platyPS/releases)
-* Optionally but recommended for development: [Visual Studio 2017](https://www.visualstudio.com/downloads/)
-
-#### Steps
-* Obtain the source
-    - Download the latest source code from the [release page](https://github.com/PowerShell/PSScriptAnalyzer/releases) OR
-    - Clone the repository (needs git)
-    ```powershell
-    git clone https://github.com/PowerShell/PSScriptAnalyzer
-    ```
-* Navigate to the source directory
-    ```powershell
-    cd path/to/PSScriptAnalyzer
-    ```
-* Building
-
-    You can either build using the `Visual Studio` solution `PSScriptAnalyzer.sln` or build using `PowerShell` specifically for your platform as follows:
-    * Windows PowerShell version 5.0 and greater
-    ```powershell
-    .\buildCoreClr.ps1 -Framework net451 -Configuration Release -Build
-    ```
-    * Windows PowerShell version 3.0 and 4.0
-    ```powershell
-    .\buildCoreClr.ps1 -Framework net451 -Configuration PSV3Release -Build
-    ```
-    * PowerShell Core
-    ```powershell
-    .\buildCoreClr.ps1 -Framework netstandard1.6 -Configuration Release -Build
-    ```
-* Build documenatation
-    ```powershell
-    .\build.ps1 -BuildDocs
-    ```
-* Import the module
-```powershell
-Import-Module .\out\PSScriptAnalyzer\PSScriptAnalyzer.psd1
-```
-
-To confirm installation: run `Get-ScriptAnalyzerRule` in the PowerShell console to obtain the built-in rules
-
-* Adding/Removing resource strings
-
-For adding/removing resource strings in the `*.resx` files, it is recommended to use `Visual Studio` since it automatically updates the strongly typed `*.Designer.cs` files. The `Visual Studio 2017 Community Edition` is free to use but should you not have/want to use `Visual Studio` then you can either manually adapt the `*.Designer.cs` files or use the `New-StronglyTypedCsFileForResx.ps1` script although the latter is discouraged since it leads to a bad diff of the `*.Designer.cs` files.
+#### Building And Testing
```

Comment:
It does not seem to me to be a sub-section of `Installation` to me. I think this should be rather have the same hierarchical level as `Installation` or `Suppressing Rules`.

Reply:
ok